sdk/logs.js

```diff
@@ -29,23 +27,17 @@ const logs = async function logs(functionName, apiKey, follow, startAfter, token
   };
 
   for (let attempt = 1, backoff = 3; attempt <= 3; attempt += 1, backoff *= 2) {
+    // eslint-disable-next-line no-await-in-loop
+    const response = await loggedRequest(options, 'get', false);
```

Comment:
1. Why not use the validated version? 
2. Why fail silently after 3 attempts and not alert the user?
(not sure about the multiple attempts in the first place, nor about the backoff ms sleep times (3,6,12) but this is another issue)

Reply:
The validated version would remove the logic you describe in point 2.

I wasn't the author of the logging logic so can't provide insight into the specific choices made. Again most changes I make are with agility in mind, the feature is to standardize the error handling/requests between SDK calls, not improve logging logic.